### PR TITLE
Improvement/catalog doc

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: docs/catalog-install.md [hunk 1]
```diff
@@ -5,37 +5,47 @@ The Service Catalog component mantains the state of the tasks submitted to the S
...
-  2. sudo apt-get install -y postgresql
+  ```bash
+  sudo apt-get update
+  sudo apt-get install -y postgresql
   ```
 
 ## Configure
 
-To configure SAPS catalog, first create an user and a database (```$catalog_user``` and ```$catalog_db_name```). You also need to define a password (```$catalog_passwd```) to access the database. See below commands:
+To configure SAPS catalog, first create an user, defined by a name (```catalog_db_name```) and password (```catalog_passwd```). In addition, create a database (```catalog_db_name```). Fill the variables and run below commands:
```
Comment (thiagoyeds) [inline docs/catalog-install.md]: Change the first ````catalog_db_name``` to ```catalog_user```

File: docs/catalog-install.md [hunk 2]
```diff
@@ -5,37 +5,47 @@ The Service Catalog component mantains the state of the tasks submitted to the S
...
+  installed_version=<postgres_version>
+  sed -i 's/peer/md5/g' /etc/postgresql/$installed_version/main/pg_hba.conf
+  bash -c 'echo "host all all 0.0.0.0/0 md5" >> /etc/postgresql/$installed_version/main/pg_hba.conf'
+  sudo sed -i "$ a\listen_addresses = '*'" /etc/postgresql/$installed_version/main/postgresql.conf
+  sudo service postgresql restart
   ```
 
 ## Test
-To test the deploy and configuration of the SAPS catalog (assuming it is accessible via the ```$catalog_ip_address``` IP address), one can try opening a conection to it from another machine (it is necessary to have PostgreSQL installed), using the below command:
+To verify whether the deploy was successful, one can try opening a conection (assuming it is accessible via the ```$catalog_ip_address``` IP address) to it from another machine (it is necessary to have PostgreSQL installed), using the below command:
```
Comment (thiagoyeds) [inline docs/catalog-install.md]: Change ```conection``` to ```connection```

**Review by thiagoyeds — CHANGES_REQUESTED** [review]
Nice improvement @wesleymonte, verify my change requests

File: docs/catalog-install.md (hunk 1 above)
Comment (wesleymonte) [reply docs/catalog-install.md]: Done.

File: docs/catalog-install.md (hunk 2 above)
Comment (wesleymonte) [reply docs/catalog-install.md]: Done.

**Review by thiagoyeds — APPROVED** [review]
Lgtm

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**